project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala

```diff
@@ -77,6 +78,8 @@ object Dependencies {
 
   val ClientMainDependencies: Seq[ModuleID] = Seq(
     "ch.qos.logback" % "logback-classic" % LogbackClassicVersion,
+    "com.github.alexarchambault" %% "case-app" % CaseAppVersion,
+    "com.iheart" %% "ficus" % FicusVersion,
```

Comment:
Nitpick: can you move this below `scopt` to keep the ABC order?

Reply:
scopt shouldn't be needed any more

Reply:
also g comes before i

Reply:
Yup was asking to move the `com.iheart` below, but moot point with scopt going away